content/800-data-platform/200-projects/110-create-new-application-from-template.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Create a new project from a template'
+metaTitle: 'Create a new Node.js application project from a template in the Prisma Data Platform'
+metaDescription: 'If you want to explore how to create projects in the Prisma Data Platform, you can start with template Prisma schema which is immediately migrated to an empty database.'
+tocDepth: 3
+---
+
+<TopBlock>
+
+If you want to explore how to create projects in the Prisma Data Platform, you can start with template Prisma schema which is immediately migrated to an empty database.
+</TopBlock>
+
+## Prerequisites and considerations
+
+* GitHub account
+* [Sign up for the Prisma Data Platform](/data-platform/accounts#sign-up-for-the-prisma-data-platform)
+* Connection string to a [Prisma ORM-supported databases](/reference/database-reference/supported-databases)
+* [Grant the Prisma GitHub app read and write permissions to your GitHub repositories](/data-platform/projects#grant-read-and-write-permissions-to-the-prisma-github-app)
+
+## Steps
+
+1. Go to the [_Projects_](https://cloud.prisma.io/projects) page and click **New Project**.
+2. On the _Configure project_ page, name your project and create a new GitHub repository.
+    1. In the **Display Name** text box, provide a name for your project.
+        <Admonition>
+        
+        **Note**<br /><br />
+        The display name of a project appears in several places in the Prisma Data Platform.<br />
+          &nbsp;• [_Projects_](https://cloud.prisma.io/projects) page<br />
+          &nbsp;• [_Billing_](https://cloud.prisma.io/billing) page<br />
+          &nbsp;• **Projects** drop-down in the upper-left
+        </Admonition>
+    2. From the **GitHub Accounts & Organizations** drop-down menu, select a GitHub account or an organization.
+    3. Select the **Create a repository** option.
+    4. (Optional) In the **Repository Name** text box, enter a name for the repository.
+
+        <Admonition type="info">
+        
+        **Note**<br /><br />
+
+        • The GitHub repository name is pre-filled based on the display name and each space is replaced with a hyphen.<br /><br />
+        • The GitHub repository name cannot duplicate other repository names.<br />
+        </Admonition>
+
+    5. Click **Next**.
+        <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1666310180/docs/pdp/create-manage-projects/create-project-01-configure-project_hpijvd.png"
+        alt="Prisma Data Platform - Manage project members - Step 03 - Click Edit for the project member you want to delete"
+        width="750px"
+        />
+3. On the _Select template_ page, select an application template for your first project.
+    1. Select one of the available application templates and preview its Prisma schema file below.
```

Comment:
Do you mean here that the UI displays the schema file lower down on the screen? Or that the user will get a chance to preview the file later in the procedure?

Edit: ah, I see from the screenshot that you mean the former. Could you say something like:

1. Select one of the available application templates. When you select it, you can see a preview of its Prisma schema file below.